addons/account/models/account_move.py

```diff
@@ -638,7 +638,7 @@ def _compute_is_being_sent(self):
                   JOIN account_move_account_move_send_rel move_send_rel
                     ON move_send_rel.account_move_id in %(move_ids)s
                    AND send.id = move_send_rel.account_move_send_id
-                 WHERE send.mode != 'done'
+                 WHERE send.mode = 'invoice_multi'
```

Comment:
I think we have another problem. SInce the exec of the wizard is failing, I think the record stays forever in the db and is processed everytime the cron is run

Reply:
Yesn't. 
The cron "only" runs one time for nothing...
What happens : 
1. We run the wizard in single mode 
2. We encounter an error, since we are in single mode we throw an error and don't reach the mode='done'.
3. We trigger the cron on this wizard
4. It runs again in the error but this time doesn't throws since it's from cron, thus reach the "done" state
5. Gets deleted on next cron.
I think this can easily be fixed by adding a condition mode == 'invoice_multi' to trigger the cron.
Will prepare a PR if you agree :)